lib/CL/devices/basic/basic.c

```diff
@@ -247,6 +263,21 @@ pocl_basic_init (unsigned j, cl_device_id device, const char* parameters)
                      | CL_DEVICE_SVM_ATOMICS;
 #endif
 
+  if (strstr (HOST_DEVICE_EXTENSIONS, "cl_intel_unified_shared_memory")
```

Comment:
Can you make this a build-time check?

Reply:
Another question is if there's a need to _not_ enable USM since it's an additional service?